Cargo.toml

```diff
@@ -232,148 +232,148 @@ datafusion = { version = "49.0.2", default-features = false, features = [
+js-sys = "0.3.81"
+jsonwebtoken = { version = "9.3", default-features = false }
 lance = { version = "0.36.0", default-features = false } # When you update this, also update the list of features enabled for `datafusion` (~50 lines up)
-libc = "0.2"
-linked-hash-map = { version = "0.5", default-features = false }
-log = "0.4"
-log-once = "0.4"
-lz4_flex = "0.11"
+libc = "0.2.176"
+linked-hash-map = { version = "0.5.6", default-features = false }
+log = "0.4.28"
+log-once = "0.4.1"
+lz4_flex = "0.11.5"
 macaw = "0.30.0"
-mcap = "0.23.1"
-memmap2 = "0.9.5"
-memory-stats = "1.1"
-mimalloc = "0.1.43"
-mime_guess2 = "2.0" # infer MIME type by file extension, and map mime to file extension
+mcap = "0.23.3"
+memmap2 = "0.9.8"
+memory-stats = "1.2"
+mimalloc = "0.1.48"
+mime_guess2 = "2.3" # infer MIME type by file extension, and map mime to file extension
 mint = "0.5.9"
 natord = "1.0.9"
-ndarray = "0.16"
-ndarray-rand = "0.15"
-never = "0.1"
-nohash-hasher = "0.2"
-notify = { version = "8", features = ["macos_kqueue"] }
-num-derive = "0.4"
-num-traits = "0.2"
-numpy = "0.24"
-objc2-app-kit = "0.3"
-opentelemetry = { version = "0.30", features = ["metrics"] }
-opentelemetry-appender-tracing = "0.30"
-opentelemetry-otlp = "0.30"
-opentelemetry_sdk = { version = "0.30", features = ["rt-tokio"] }
-ordered-float = "4.3.0"
-parking_lot = "0.12.3"
-parquet = { version = "55.1", default-features = false }
+ndarray = "0.16.1"
+ndarray-rand = "0.15.0"
+never = "0.1.0"
+nohash-hasher = "0.2.0"
+notify = { version = "8.2", features = ["macos_kqueue"] }
+num-derive = "0.4.2"
+num-traits = "0.2.19"
+numpy = "0.24.0"
+objc2-app-kit = "0.3.2"
+opentelemetry = { version = "0.30.0", features = ["metrics"] }
+opentelemetry-appender-tracing = "0.30.1"
+opentelemetry-otlp = "0.30.0"
+opentelemetry_sdk = { version = "0.30.0", features = ["rt-tokio"] }
+ordered-float = "4.6.0"
+parking_lot = "0.12.5"
+parquet = { version = "55.2", default-features = false }
 paste = "1.0"
-pathdiff = "0.2"
+pathdiff = "0.2.3"
 percent-encoding = "2.3"
-pico-args = "0.5"
-pin-project-lite = "0.2"
-ply-rs = { version = "0.1", default-features = false }
-poll-promise = "0.3"
-pollster = "0.4"
-prettyplease = "0.2"
+pico-args = "0.5.0"
+pin-project-lite = "0.2.16"
+ply-rs = { version = "0.1.3", default-features = false }
+poll-promise = "0.3.0"
+pollster = "0.4.0"
+prettyplease = "0.2.37"
 proc-macro2 = { version = "1.0", default-features = false }
-profiling = { version = "1.0.12", default-features = false }
-prometheus-client = "0.24"
-prost = "0.13.3"
-prost-build = "0.13.3"
+profiling = { version = "1.0.17", default-features = false }
+prometheus-client = "0.24.0"
+prost = "0.13.5"
+prost-build = "0.13.5"
 prost-reflect = "0.15.3"
-prost-types = "0.13.3"
-protoc-prebuilt = "0.3"
+prost-types = "0.13.5"
+protoc-prebuilt = "0.3.0"
 puffin = "0.19.1"
-puffin_http = "0.16"
-pyo3 = "0.24.1"
-pyo3-build-config = "0.24.1"
+puffin_http = "0.16.1"
+pyo3 = "0.24.2"
+pyo3-build-config = "0.24.2"
```

Comment:
these always make me nervous since is part of the pyo3-arrow-dfn-lance gang. but maybe wrongly so?